docusaurus/docs/guides/deployments/10-linux/20-router/50-migrate.mdx

```diff
@@ -3,48 +3,38 @@ title: Migrate a Router Installation
 sidebar_label: Migrate
 ---
 
-You can generate a configuration with the `ziti create config router` command, optionally mutating the generated config with a combination of command-line args and environment variables. Find an annotated sample config file from [the Ziti repo](https://github.com/openziti/ziti/blob/main/etc/edge.router.yml).
+Here's an example of migrating an existing router configuration to the Linux service's working directory.
 
-Here's an example BASH script for migrating an existing controller state to the Linux service's working directory.
+1. Ensure service is disabled and state is clean.
+    
+    ```text
+    sudo systemctl disable --now ziti-router.service
+    sudo systemctl clean --what=state ziti-router.service
+    ```
 
-```bash
-#!/bin/bash
+1. Duplicate the router-related parts of the configuration to the service's working directory.
```

Comment:
though Duplicate is a synonym for copy, "copy" is a much more natural sounding word here

Reply:
Improved in https://github.com/openziti/ziti-doc/pull/868